app/js/common.js

```diff
@@ -0,0 +1,7 @@
+var isAvailable = function(team) {
+	return team.currentTeamSize < team.size;
+}
+
+var getAvailableTeam = function (teams) {
```

Comment:
add documentation comment
`/**
 * filter teams that still have places left
 * @param teams array of teams
 * @return filtered teams
 */